src/main/resources/assets/computercraft/lua/rom/programs/shell.lua

```diff
@@ -114,7 +115,22 @@ else
 end
 
 local function run( _sCommand, ... )
-    local sPath = shell.resolveProgram( _sCommand )
+    local checkvar = { ... }
+    table.insert( checkvar, 1, _sCommand )
```

Comment:
Let's not let you use a variable as the command name, that's what aliases are for.